01/jeus.js

```diff
@@ -0,0 +1,67 @@
+const discounts = [10, 20, 30, 40]
+// 1. 쿠폰과 할인율 모든 경우의 수 구하기
+// 2. 모든 경우의 수를 이용해 가격 유저가 해당 할인율 보다 많이 쓰는 경우 계산
+function solution(users, emoticons) {
+    const allCases = []
+    let plusUser = 0; // 플러스 가입자수
+    let salesAmount = 0; // 판매액
+
+    getAllCases(emoticons, [], allCases)
```

Comment:
p4;
call-by-reference보다는 함수에서 반환받는것도 깔끔할것 같아요..!
```suggestion

    let plusUser = 0; // 플러스 가입자수
    let salesAmount = 0; // 판매액

    const allCases = getAllCases(emoticons, [])
```

Reply:
좀더 가독성이 깔끔할것같군요. 수정해둘게요!